virttest/utils_misc.py

```diff
@@ -536,10 +536,13 @@ def get_path(base_path, user_path):
     :param base_path: The base path of relative user specified paths.
     :param user_path: The user specified path.
     """
-    if os.path.isabs(user_path) or aurl.is_url(user_path):
+    if aurl.is_url(user_path):
         return user_path
```

Comment:
sorry, i didn't make it clear, i was commenting on suqinhuang's comment.(seems can delete these two lines)